enrichment-common/src/main/java/eu/europeana/enrichment/common/commons/EnrichmentConstants.java

```diff
@@ -91,5 +91,9 @@ public class EnrichmentConstants {
 	public static final String LATITUDE_JSONPROP = "claims.P625.mainsnak.datavalue.value.latitude";
 	public static final String LONGITUDE_JSONPROP = "claims.P625.mainsnak.datavalue.value.longitude";
 	public static final String INSTANCE_OF_JSONPROP = "claims.P31.mainsnak.datavalue.value.id";
+	
+	// Types 
+	public static final String TOPIC_TYPE = "Topic";

```

Comment:
As I previously said, there is a class EntityTypes for the types. In general, first we need to check if there is already existing part of the code that can be reused for the new functionalities and which part fits the best to be reused, then add a new code, not just add the new code anywhere.